### ISSUE TITLE
Keycloak Admin console not appearing

### ISSUE THREAD

**Issue by mamahara** [keycloak/keycloak#12684]
### Describe the bug

I have done Keycloak 18 installation on Linux machine and starting the server as below.

kc.sh start -Dkeycloak.profile.feature.upload_scripts=enabled -Dkeycloak.profile.feature.scripts=enabled --auto-build

Server is starting properly but after login the admin console does not show completely.
When i start server adding keycloak.profile as preview , admin console is appearing.

How does the old admin console will appear in keycloak 18.

### Version

18.0.0

### Expected behavior

Admin console should show after login

### Actual behavior

Admin console does not show after login and only keycloak banner is appeared

**Comment by bz0qyz** [comment]
+1 on this issue. I am seeing the same thing on 18.0.2

**Comment by pedroigor** [comment]
When using the `start` command it is expected that you have TLS enabled. Otherwise, the server URLs are built using the `https` scheme and the default TLS port (e.g.: `8443`). As a result you have a blank page and the page source like this:

```
<script type="text/javascript">
        var authServerUrl = 'https://localhost:8443';
        var authUrl = 'http://localhost:8080';
        var consoleBaseUrl = '/admin/master/console/';
        var resourceUrl = '/resources/3op7l/admin/keycloak';
        var masterRealm = 'master';
        var resourceVersion = '3op7l';
    </script>
```

Not the URL for the `authServerUrl` above, it won't work because you are not listening on `8443`.

You can try to use the `hostname-strict-https=false` option to not force the `https` scheme when running in production mode (e.g.: when running the `start` command).

**Comment by pedroigor** [comment]
We introduced the `hostname-url` and `hostname-admin-url` options and they should be available in `19.0.2`.

Basically, they keep the same behavior as before when using the legacy distribution so that you can provide a base URL for both frontend and admin URLs.

**Comment by adamo901** [comment]
How to use theme please?
I am facing the same issue with 19.0.2.
